Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ActionButton from './components/ButtonComponents/ActionButton';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="calc-container">
+      <CalculatorDisplay />
+      <ActionButton text="clear" />
+      <NumberButton buttonStyle="operator" text="&divide;" />
+      <NumberButton buttonStyle="number" text="7" />
+      <NumberButton buttonStyle="number" text="8" />
+      <NumberButton buttonStyle="number" text="9" />
+      <NumberButton buttonStyle="operator" text="&times;" />
+      <NumberButton buttonStyle="number" text="4" />
+      <NumberButton buttonStyle="number" text="5" />
+      <NumberButton buttonStyle="number" text="6" />
+      <NumberButton buttonStyle="operator" text="-" />
+      <NumberButton buttonStyle="number" text="1" />
+      <NumberButton buttonStyle="number" text="2" />
+      <NumberButton buttonStyle="number" text="3" />
```

Comment:
You could automate the construction of most of these buttons. 

You could write an array of numbers.  Create a `numPad` sub-components and map through that array returning a new button with style set to number and the text to the number passed into your call back by the `.map` method. Give that a width and flex row.